tests/feed.rs

```diff
@@ -298,3 +298,36 @@ async fn audit_bad_data() {
         }
     }
 }
+
+#[async_std::test]
+async fn try_open_missing_dir() {
+    use rand::{thread_rng, Rng};
+    use rand::distributions::Alphanumeric;
+
+    let rand_string: String = thread_rng()
+        .sample_iter(&Alphanumeric)
+        .take(5)
+        .collect();
+
+    let path = format!("some_nonexistent_dir/{}", rand_string);
+
+    let feed = Feed::open(path).await;
+    match feed {
+        Ok(_) => {
+            panic!("Opening nonexistent path should result in error");
+        }
+        Err(_) => { }
+    }
+}
```

Comment:
This can be written more succinctly as:

```suggestion
    if Feed::open(path).await.is_ok() {
        panic!("Opening nonexistent path should result in error");
    }
```

The same applies for the next code block over.